src/app/SelectOverrideDialog.qml

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2013-2016 Canonical Ltd.
+ * Copyright 2016-2020 Ubports
```

Comment:
@UniversalSuperBox can "Ubports" be used as copyright owner by now ?

Reply:
"UBports Foundation" is more correct, but valid